### ISSUE TITLE
color not added to tab

### ISSUE THREAD

**Issue by monkeydri** [mozilla/multi-account-containers#1018]
I just downloaded this add-on, tried both the Firefox add-ons xpi and the latest Github release xpi.

I chose a tab, click the extension button, and select a container (ex work or personal) but nothing happens, tab is not getting colored.

- Multi Account Containers 4.1.0
- Ubuntu 17.10
- Firefox Quantum 57.0 (64-bit) - standard theme

**Comment by groovecoder** [comment]
Can you include a screenshot where you don't see the color?

**Comment by monkeydri** [comment]
The problem was that I misunderstood how this add-on worked : it creates a new tab in a container whereas I thought it would put the currently selected tab in the container.

Because I was using a custom "new tab" replacement the new tab did not appear so I could not see the underline of the tab and the colored tag in the URL bar. Now that I switched back to a more classic "new tab" behavior it works correctly. Unfortunately I cannot reproduce the issue I had as I forgot which add-on or about:config trick I used at the time (I tried many...).

Anyway this add-on is amazing ! I just have to find a way to make it work with [Vertical tabs reloaded](https://addons.mozilla.org/en-US/firefox/addon/vertical-tabs-reloaded/) add-on now.
